recipes/CRYSTALpytools/LICENSE.txt

```diff
@@ -0,0 +1,7 @@
+The MIT License (MIT) Copyright (c) 2011-2012 MIT & The Regents of the University of California, through Lawrence Berkeley National Laboratory
```

Comment:
The file is still here. Can you please remove it before marking as resolved? We prefer to use the tarball version to avoid forking from the original license in case the file is updated upstream.